### DISCUSSION TITLE
Feature Request: Better support for cell options in vs code

### DISCUSSION THREAD

**Discussion by travis-leith** [quarto-dev/quarto-cli#10590] [Feature Requests]
Sorry if this is discussed elsewhere; it is difficult to search for this.

I would like to have better options in vs code for configuring things like `#| echo: false` in the cells. 
As far as I can tell, I need to do this manually and I need to know in advance what all the options are (or consult a reference).

In RStudio, using RMardown, there is a little gear icon I can click and then select how I want the cell to behave. I am not aware of anything like this for Quarto.

**Comment by cscheid** [comment]
This is a feature request, so I'm moving it to our discussions forum.
